basic_auth.go

```diff
@@ -34,6 +35,10 @@ var (
 		"Path to YAML file with server_user, server_password keys for HTTP Basic authentication "+
 			"(overrides HTTP_AUTH environment variable).",
 	)
+	authFileFKingpin = kingpin.Flag(
+		"web.auth-file",
+		"Path to YAML file with server_user, server_password keys for HTTP Basic authentication "+
+			"(overrides HTTP_AUTH environment variable).").Default("").String()
```

Comment:
Drop original `authFileF`, rename `authFileFKingpin` to `authFileF`. Do the same for all flags.
We have tagged release with `flag` package, so we could safely drop it in the `master` version.
In `node_exporter`, use `master` version for development. When it works, poke me, I will make a new tag for `exporter_shared`, and we will use it.

Reply:
Done